lib/commons/color/element-has-image.js

```diff
@@ -0,0 +1,36 @@
+/* global color */
+
+/**
+ * Reports if an element has a background image or gradient
+ *
+ * @method elementHasImage
+ * @memberof axe.commons.color
+ * @private
+ * @param {Element} elm
+ * @param {Object|null} style
+ * @return {Boolean}
+ */
+color.elementHasImage = function elementHasImage(elm, style) {
+	const graphicNodes = ['IMG', 'CANVAS', 'OBJECT', 'IFRAME', 'VIDEO', 'SVG'];
+	const nodeName = elm.nodeName.toUpperCase();
+
+	if (graphicNodes.includes(nodeName)) {
+		axe.commons.color.incompleteData.set('bgColor', 'imgNode');
```

Comment:
Missing tests for imcompleteData.

Reply:
See - https://github.com/dequelabs/axe-core/pull/1451/files#diff-e4959c7e9e75633a85787a47a13706b0R24